index.html

```diff
@@ -10,10 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    
+    <main>
+        <header class="header">
+            <img class="fav_ico" src="img/karma-logo.svg"/>
+            
+            <div>          
+                <nav class="nav_ul">
+                    <a href="#">Meet Karma</a>
+                    <a href="#">How it Works</a>
+                    <a href="#">Store</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Help</a>
+                    <a href="#">Login</a>
+                </nav>
+            </div>
+            
+        </header>
+    </main>    
+    <main class="main_back_img">
+        <div class="ad_back_img">   
+            <h3>introducing Karma</h3>
```

Comment:
This should probably be a `h2`, because there's no `h2` for this to be a sub-heading of. See https://syllabus.codeyourfuture.io/html-css/week-1/lesson#text-content

The `h4` below probably doesn't need to be a heading at all - it can be a paragraph (`<p>`) or `<span>` and you can style it to be the right font size, etc.

See https://www.w3schools.com/html/html_headings.asp for an overview of how and when to use headings, and why it's important to only use them when the text really is a heading (and not just "big text").